pkgs/by-name/sy/syncyomi/package.nix

```
@@ -0,0 +1,106 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, buildGoModule
+, jq
+, moreutils
+, nodePackages
+, cacert
+, esbuild
+}:
+
+buildGoModule rec {
+  pname = "syncyomi";
+  version = "0.3.8";
+
+  src = fetchFromGitHub {
+    owner = "SyncYomi";
+    repo = "SyncYomi";
+    rev = "v${version}";
```

```suggestion
    rev = "refs/tags/v${version}";
```

prevents the update bots from confusing branches vs tags